subprojects/docs/docs.gradle

```diff
@@ -93,6 +102,8 @@ dependencies {
 ext {
     srcDocsDir = file('src/docs')
     userguideSrcDir = new File(srcDocsDir, 'userguide')
+    userguideAsciidocGeneratedDir = new File(buildDir, 'asciidoc/userguide')
```

Comment:
I always have doubts: will this work under Windows? Say, compared to `file("$buildDir/asciidoc/userguide")`?